app/console

```diff
@@ -29,5 +31,15 @@ if ($debug) {
 }
 
 $kernel = new AppKernel($env, $debug);
+
+// make our container available statically. This is needed to make our static models work
+$kernel->boot();
+$kernel->defineForkConstants();
+BaseModel::setContainer($kernel->getContainer());
+
+define('APPLICATION', 'Console');
```

Comment:
@WouterSioen We must define "APPLICATION" here. Otherwise we get the following error:
```php
PHP Notice:  Use of undefined constant APPLICATION - assumed 'APPLICATION' in ./src/Backend/Init.php on line 92
```
But I'm not sure if we should use `Backend` as value, or something new like for example: `Console`